cmd/miner/symbols/lib/robot.go

```diff
@@ -257,18 +257,16 @@ func (this *QitmeerRobot) SubmitWork() {
 							this.InvalidShares++
 						}
 					}
-					this.SubmitLock.Unlock()
```

Comment:
why miner is changed in the wallet module? Please separate to another PR to clarify if the related lock mechanism really has to be changed or any bug here need to be fixed.